proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1186,7 +1186,7 @@ public SymbolScopeSuper call() throws Exception {
     // We take a copy of the cached superScope, because the tree parser messes with
     // the attributes of the symbols, and we don't want to mess with the symbols that
     // are in the super scopes in the cache.
-    rootScope.assignSuper(cachedCopy.generateSymbolScopeSuper());
+    // rootScope.assignSuper(cachedCopy.generateSymbolScopeSuper());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)